Documentation/configuration/api-restrictions.rst

```diff
@@ -0,0 +1,37 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+*****************************
+Administrative API Enablement
+*****************************
+
+Cilium 1.14 introduced a new set of flags that may be used to selectively
+enable which API endpoints are exposed to clients. When an API client makes a
+request an API endpoint that is administratively disabled, the server responds
+with an HTTP 403 forbidden error.
+
+The option may be configured with a list of endpoints as described below, or
+alternatively, by providing an option with the ``*`` suffix. If ``*`` is
+provided directly as a flag value, then all APIs are enabled. If there is
+text before the ``*``, then the API flag must start with that prefix in order
+for the flag to enable that option. For instance, ``Get*`` enables all
+read-only "GET" APIs without enabling any write APIs.
+
+The cilium-agent relies on several of these APIs for its basic duties. In
+particular, if the following APIs are disabled then it is likely to cause
+significant disruption to the operations of the agent:
```

Comment:
clarity, active voice

One of the rare times when future tense is appropriate 🦄 
```suggestion
The cilium-agent relies on several of these APIs for its basic duties. In
particular, disabling the following APIs will likely cause
significant disruption to agent operations:
```